app/api/custom/events.py

```diff
@@ -30,17 +30,18 @@ def get_dates(event_id):
 @jwt_required
 def contact_organizer(event_id):
     event = Event.query.get_or_404(event_id)
-    organizers_emails = list(map(lambda x: x.email, event.organizers)) + list(map(lambda x: x.email, event.coorganizers))
+    organizers_emails = set(list(map(lambda x: x.email, event.organizers)) + list(map(lambda x: x.email, event.coorganizers)))
     context = {
         'attendee_name': current_user.fullname,
         'attendee_email': current_user.email,
         'event_name': event.name,
+        'email_copy': request.json.get('email'),
     }
     organizer_mail = (
         "{attendee_name} ({attendee_email}) has a question for you about your event {event_name} </br>"
         + request.json.get('email')
     )
-    attendee_mail = "Hello, you have contacted the organizers of the event {event_name}. Below you find a copy of your email. Organizers have received your message and will follow up with you. This is a system message. Please do not reply to this message. Replies are not monitored. Thank you."
+    attendee_mail = "Hello, you have contacted the organizers of the event {event_name}. Below you find a copy of your email. Organizers have received your message and will follow up with you. This is a system message. Please do not reply to this message. Replies are not monitored. Thank you. <br/> {email_copy}"
```

Comment:
This is what the email looks like

![image](https://user-images.githubusercontent.com/3874064/105223958-84fd4900-5b82-11eb-8668-faf3fab022f3.png)

As we progress as developers, our goal is not just to check list of items to be completed for the issue to be closed but to also think is the end result actually provides a valid user experience? 
The message is not readable, even Thank You is not on the next line.

Secondly, I don't think you even tested the mail, because there is no variable named `email_copy`, this will definitely not work.